src/resource.ts

```diff
@@ -503,7 +503,7 @@ export class BaseResource implements IResource {
       offset: (pagination.page - 1) * pagination.perPage,
       first: pagination.perPage,
       filter: filters,
-      orderBy,
+      orderBy: filterOrderBy.concat(orderBy),
```

Comment:
shouldn't we put orderBy first and add filterOrderBy then.
otherwise i think sorting in listings will be broken as soon as we use a fulltext-filter.

example usecase: search for companies starting with "Space*" and then ordering by their state.
(they will be sorted by ft-rank and then state instead of state and if the same state by ft-rank)

Reply:
There is always a default order by filtering (e.g. created by). If we prefer this one we will never get the sorting by rank and can skip it totally. But I thinbk if we use a search we want the rank of the search to speak.

Reply:
Currently it's not possible to set a filter and sort simultaneously, which makes it hard to implement it. If in future this is possible I kept the revert commit of this change here: https://github.com/lovelysystems/ra-data-postgraphile/tree/burnes/revert-auto-ordering